crates/wasmtime/src/runtime/store.rs

```diff
@@ -794,6 +796,9 @@ impl<T> Store<T> {
     /// Note that it is not required to actively call this function. GC will
     /// automatically happen when internal buffers fill up. This is provided if
     /// fine-grained control over the GC is desired.
+    ///
+    /// This method is only available when the `gc` Cargo feature is enabled.
```

Comment:
You can actually skip this documentation as rustdoc will auto-generate it with the right annotations. Elsewhere in the codebase we've got `#[cfg_attr(docsrs, doc(cfg(feature = "async")))]`, so could all public methods gated behind the `gc` feature have a similar annotation?

Reply:
It requires the `doc_cfg` nightly feature for rustdoc to add that annotation, which `docs.rs` enables but local `cargo doc` invocations probably don't. Therefore, I think it is worth leaving the comment there.

Reply:
True yeah, but that's why it's gated by `docsrs` so the docs.rs docs are the only ones that show this. I'd recommend using the `#[cfg_attr]` approach since that's what everything else does in the API docs right now, or otherwise we should remove all those and add manual docs there too